src/libraries/System.Security.Cryptography.Algorithms/tests/AesCcmTests.cs

```diff
@@ -690,4 +675,21 @@ public static IEnumerable<object[]> GetNistCcmTestCasesWithNonEmptyPT()
             },
         };
     }
+
+    public class AesCcmIsSupportedTests
+    {
+        [Fact]
+        [SkipOnPlatform(TestPlatforms.Browser, "This test runs only on non-Browser.")]
+        public static void IsSupported_NotOnBrowser_ReturnsTrue()
+        {
+            Assert.True(AesCcm.IsSupported);
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Browser)]
+        public static void IsSupported_OnBrowser_ReturnsFalse()
+        {
+            Assert.False(AesCcm.IsSupported);
+        }
```

Comment:
Feels like it's just

```C#
public static void CheckIsSupported()
{
    bool shouldWork =!Environment.IsBrowser();

    if (shouldWork)
    {
        Assert.True(AesCcm.IsSupported);
    }
    else
    {
        Assert.False(AesCcm.IsSupported);
    }
}
```

That way we don't have to have redundant platform statements on the true and false tests.